report-generator/compile_latency.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+
+LATENCY_ADOC_FILE=$(readlink -f "include/latency-test-reports.adoc")
+STATS_FILE="include/stats_results"
+
+generate_latency_results()
+{
+    # Generate graph and stats associated to a publisher and subscriber test
+    # Parameters: $1: publisher result file
+    # $2: subscriber result file
+
+    local PUB_DATA_FILE=$1
+    local SUB_DATA_FILE=$2
+    python3 timestamp_analysis/timestamp_analysis.py \
+    $PUB_DATA_FILE \
+    $SUB_DATA_FILE >> $STATS_FILE
```

Comment:
rm $STATS_FILE at the beginning of the script to avoid importing previous aborted tests